README.md

```diff
@@ -2,7 +2,16 @@
 
 This is my first attempt at building, myself, a full-stack JS application using a relational database. We are using node, express and postgres -- all containerized through docker.
 
-## Resources I've used so far
+# Setup Guide
+
+You will need a few things:
+- Docker engine: https://docs.docker.com/engine/install/
+- Docker compose: https://docs.docker.com/compose/install/
+    - If you're on Linux, https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user will come in very handy
+- Environment files: Ask Jordyn for now since they have these in the form that works best
```

Comment:
Refer to `{app}.{environment}.env` but also is there a better and less painful way of doing envs? 🤔🤔🤔

Reply:
The reason I put this here is because there are the samples but they are empty. I would like to see some dummy data there since it would help and could make the samples become tests right out of the box. If you're developing then it is easy-peasy to get setup.

Overall I would like to keep our environment variables in the cloud and have API access to get them for everyday use. I looked into it this morning and generally keeping them in a separate system is the common practice. I like the dotenv tool and would like to use that but up the security of where they are coming from

Reply:
FTR I added an issue which I'll address 🔜 

https://github.com/JordynMarcellus/node-postgres-experiment/issues/4